pokeworld.py

```diff
@@ -6,12 +6,12 @@ def deepcopy(actualAttack):
     return newattack
 
 small_pokemons = ['pikachu', 'charmander', 'squirtle', 'bulbasaur', 'caterpie', 'weedle', 'pidgey', 'ekans', 
-                  'sandshrew', 'nidoran',  'vulpix', 'jigglypuff', 'zubat',  'diglett', 'meowth', 'psyduck',  
+                  'sandshrew', 'nidoran', 'clefairy', 'vulpix', 'jigglypuff', 'zubat',  'diglett', 'meowth', 'psyduck',  
                   'growlithe', 'abra', 'geodude', 'shellder', 'gastly', 'onix', 'koffing', 'horsea', 'staryu', 'scyther', 'magmar', 'magikarp', 
                   'eevee']
 
 medium_pokemons = ['raichu', 'charmeleon', 'wartortle', 'ivysaur', 'metapod', 'kakuna', 'pidgeotto', 'arbok', 'sandslash', 
-                   'nidorina', 'nidorino', 'ninetales',  'golbat',  'dugtrio', 
+                   'nidorina', 'nidorino', 'clefable', 'ninetales',  'golbat',  'dugtrio', 

```

Comment:
There are some conflicts in the merging. I guess it is due to merging cubone and marowak which are not in this list. Wait let me try to update them here

Reply:
Ah, sorry for the inconvenience! It seems you fixed it anyway? :-D

Reply:
Ya just a few changes and it was good to go